server/controllers/refer/courseParticipationsController.ts

```diff
@@ -18,4 +58,24 @@ export default class CourseParticipationsController {
       })
     }
   }
+
+  private async courseParticipationsWithNames(
```

Comment:
Again, we'll probably be able to get rid of this soon when we move the "name", but it does the job.